### ISSUE TITLE
Triple buffering

### ISSUE THREAD

**Issue by jan-kleks** [torkel104/libstrangle#22]
Are there any settings for triple buffering?

**Comment by torkel104** [comment]
I think that's a driver setting.

**Comment by jan-kleks** [comment]
@torkel104 Yeah, it often is. But for instance, for Windows there is this app called [D3DOverrider](https://community.pcgamingwiki.com/files/file/84-d3doverrider/) which can force triple buffering, I don't know how much it is driver dependent.
